gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/db/FabricDatabase.java

```diff
@@ -616,25 +650,26 @@ private void postCreateDDLReplay(final EmbedConnection embedConn,
     final Statement stmt = embedConn.createStatement();
 
     try {
-      while (maxIterations-- > 0) {
-
-        // For the last iteration take the DD read lock to force any
-        // in progress DDLs to flush and avoid missing them.
-        // This alongwith the DD read lock in GfxdDDLRegion#chunkEntries
-        // ensures that all pending DDLs that have possibly not sent
-        // the GfxdDDLMessage are flushed.
-
-        ddReadLockAcquired = false;
-        if (maxIterations == 0) {
-          // pass TC as null to avoid check of DDL replay in progress in
-          // the lock method
-          // try to acquire DD lock in loop checking whether the skip lock
-          // flag has been set
-          ddReadLockAcquired = this.dd.lockForReadingInDDLReplayNoThrow(
-              this.memStore, Long.MAX_VALUE / 2, true);
-        }
-        this.memStore.acquireDDLReplayLock(true);
-        acquiredReplayLock = true;
+      // commenting out for snap-585
+//      while (maxIterations-- > 0) {
+//
+//        // For the last iteration take the DD read lock to force any
+//        // in progress DDLs to flush and avoid missing them.
+//        // This alongwith the DD read lock in GfxdDDLRegion#chunkEntries
+//        // ensures that all pending DDLs that have possibly not sent
+//        // the GfxdDDLMessage are flushed.
+//
+//        ddReadLockAcquired = false;
+//        if (maxIterations == 0) {
+//          // pass TC as null to avoid check of DDL replay in progress in
+//          // the lock method
+//          // try to acquire DD lock in loop checking whether the skip lock
+//          // flag has been set
+//          ddReadLockAcquired = this.dd.lockForReadingInDDLReplayNoThrow(
+//              this.memStore, Long.MAX_VALUE / 2, true);
+//        }
+//        this.memStore.acquireDDLReplayLock(true);
+//        acquiredReplayLock = true;
```

Comment:
Sorry to nag about this, but I will prefer a block comment here instead of //. Helps keep the older blame history.